qa/tasks/ceph_manager.py

```diff
@@ -1375,6 +1374,17 @@ def tmp(x):
             self.log = tmp
         if self.config is None:
             self.config = dict()
+
+        self.testdir = teuthology.get_testdir(self.ctx)
```

Comment:
Nothing uses this except the code below. Convert to local?

Reply:
The class had few calls to `get_testdir()`, replaced them with this class variable.